stake-pool/cli/src/main.rs

```diff
@@ -688,20 +689,22 @@ fn command_update(config: &Config, stake_pool_address: &Pubkey) -> CommandResult
         )?);
     }
 
-    println!("Updating stake pool...");
     instructions.push(spl_stake_pool::instruction::update_stake_pool_balance(
         &spl_stake_pool::id(),
         stake_pool_address,
         &stake_pool.validator_list,
     )?);
 
-    let mut transaction =
-        Transaction::new_with_payer(&instructions, Some(&config.fee_payer.pubkey()));
+    // TODO: A faster solution would be to send all the `update_validator_list_balance` instructions concurrently
+    for instruction in instructions {
+        let mut transaction =
+            Transaction::new_with_payer(&[instruction], Some(&config.fee_payer.pubkey()));
```

Comment:
I was hoping we could chunk out the instructions to fit more than one per transaction, but we can save that for another PR.

Reply:
Yes definitely.  I just didn't want to get ahead of  https://github.com/solana-labs/solana-program-library/issues/1567 here, figured we'd optimize the transaction backing later